React/CoreModules/RCTActionSheetManager.mm

```diff
@@ -198,14 +233,30 @@ - (void)presentViewController:(UIViewController *)alertController
       [actionButton setValue:cancelButtonTintColor forKey:@"titleTextColor"];
     }
     [alertController addAction:actionButton];
+#else // [macOS
+    if (index == cancelButtonIndex) {
+      // NSMenu doesn't need a cancel button, you can just click outside the menu
+      continue;
+    }
+
+    NSMenuItem *item = [[NSMenuItem alloc] initWithTitle:option action:@selector(menuItemDidTap:) keyEquivalent:@""];
```

Comment:
if client was trying to share code between mac and ios and had stuff in the destructivebuttonindices, will it still show in this regular item? or am i reading this code wrong. who feeds the destructivebuttonindicies?

Reply:
Menu items with destructive button indices will show up as regular menu items with no special styling. If it helps, I can update the test plan to show iOS and macOS side by side for the same tests.